OCPBUGS#9406: Use the unique combination of labels when configuring anti-affinity rules (#58975)
openshift/openshift-docs: modules/nodes-scheduler-pod-affinity-about.adoc

For example, using affinity rules, you could spread or pack pods within a service or relative to pods in other services. Anti-affinity rules allow you to prevent pods of a particular service  from scheduling  on the same nodes as pods of another service that are known to interfere with the performance of the pods of the first service. Or, you could spread the pods of a service across nodes, availability zones, or availability sets to reduce correlated failures.

[NOTE]
====

> A label selector might match pods with multiple pod deployments. To avoid this, you can use the unique combination of labels when configuring anti-affinity rules.

Reviewer: Just a suggestion. Feel free to take it or leave it.
```suggestion
A label selector might match pods with multiple pod deployments. Use unique combinations of labels when configuring anti-affinity rules to avoid matching pods.
```
